openvdb_houdini/SOP_OpenVDB_Activate.cc

```diff
@@ -493,14 +493,20 @@ template <typename GridType>
 static void
 sopDilateVoxels(GridType& grid, int count)
 {
-    openvdb::tools::dilateVoxels(grid.tree(), count);
+    openvdb::tools::dilateActiveLeafValues(grid.tree(), count);
 }
 
 template <typename GridType>
 static void
 sopErodeVoxels(GridType& grid, int count)
 {
-    openvdb::tools::erodeVoxels(grid.tree(), count);
+    openvdb::tools::erodeActiveLeafValues(grid.tree(), count);
+    if (grid.getGridClass() == openvdb::GRID_LEVEL_SET) {
+        openvdb::tools::pruneLevelSet(grid.tree());
+    }
+    else {
+        openvdb::tools::pruneInactive(grid.tree());
+    }
```

Comment:
@jmlait wasn't too sure what to do here. The VDB Activate SOP used to call `erodeVoxels` which always calls `pruneLevelSet`. If the grid is not marked as a level set (but really, it's if the grid has a negative background value), the VDB Activate SOP will fail with an error if you try to expand with negative values (i.e. erode). To solve this and maintain compatible behaviour, I've just branched on the grid type. However...this SOP also has a "Prune Tolerance" parameter which also calls "prune". I Could remove this branching in favour of that parameter, but it will change behaviour for anyone who is eroding and has the "Prune Tolerance" disabled.

Another question, expansion currently only expands leaf nodes (due to the old impl of dilateVoxels and erodeVoxels). I could update this to handle tiles with the new methods now? Again, it would change behaviour.